lib/presentation/engage/main/widgets/progress_widget.dart

```diff
@@ -142,3 +148,30 @@ class ProgressError extends StatelessWidget {
     );
   }
 }
+
+class ProgressWidgetTitle extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<BibleSeriesBloc, BibleSeriesState>(
+      listener: (context, state) {},
+      builder: (context, state) {
+        if (state is RecentBibleSeries) {
+          return Center(
+            child: RichText(
+                textAlign: TextAlign.center,
+                text: TextSpan(
+                    style: getIt<TextFactory>().liteTextStyle(),
+                    children: [
+                      TextSpan(text: "Your Progress for "),
+                      TextSpan(
+                          text: state.bibleSeriesList[0].title,
```

Comment:
We might have an issue here. For admins, it could be different from regular users based on your other change above. We would ideally need a function to get the latest `is_active` bible series from firestore.